### DISCUSSION TITLE
自定义单元格 CustomDataCell 函数报错 Cannot read properties of undefined (reading 'theme')

### DISCUSSION THREAD

**Discussion by flypxx** [antvis/S2#2747] [Show and tell]
报错截图：
![image](https://github.com/antvis/S2/assets/20834750/e925ede6-ca2a-4b4e-b003-d3aad26981c0)
自定义单元格写法：
![image](https://github.com/antvis/S2/assets/20834750/da7f4e56-fb8b-4083-8027-f81bc24cafd1)

排查后发现新版需要这么写:
![image](https://github.com/antvis/S2/assets/20834750/7673e947-1935-4f68-bbc9-f3ac1e73d13d)
官方文档的示例已经不能直接使用了，文档示例截图：
![image](https://github.com/antvis/S2/assets/20834750/dccac027-6fc6-45b8-90cf-250036e33ed3)
文档补充：
![image](https://github.com/antvis/S2/assets/20834750/0b87c360-b2bf-437c-b6cd-d3dde7a5c37c)

**Comment by lijinke666** [comment]
感谢反馈, 近期更新下示例

**Comment by lijinke666** [comment]
修复 PR  https://github.com/antvis/S2/pull/2748
